### ISSUE TITLE
"Go To..." window doesn't update when left open

### ISSUE THREAD

**Issue by sasumner** [notepad-plus-plus/notepad-plus-plus#8797]
### Description of the Issue
The _Go To..._ window for jumping to a line or position is "modeless" and can remain open while the editing of a document is taking place.  However, its information doesn't update as the user navigates the document.

### Steps to Reproduce the Issue
1. Open the _Go To..._ window by pressing Ctrl+g; observe:
![image](https://user-images.githubusercontent.com/30118311/91982926-f1250080-ecf8-11ea-8646-05853529c080.png)
2. Move input focus back to an editing tab (don't close the _Go To..._ window!)
3. Do some editing and/or caret navigation in the document tab

### Expected Behavior
Since _Go To..._ was left open, I expect its information to update to reflect the current state.

### Actual Behavior
_Go To..._ window remains static, presenting old information from the time it was invoked.

### Additional

Current "line" data is maintained on the N++ main status bar:
![image](https://user-images.githubusercontent.com/30118311/91983132-3fd29a80-ecf9-11ea-93ec-d20003b216ff.png)

If PR #8780 is adopted, perhaps this issue becomes moot as it will cause a display of "offset" ("position") on the status bar as well.  If this happens, perhaps the _Go To..._ window should be made "modal" as it has no value in being modeless.

### Debug Information
Notepad++ v7.8.9   (32-bit)
Build time : Jul 15 2020 - 20:26:50
Path : C:\\...............\npp.7.8.9.bin\notepad++.exe
Admin mode : OFF
Local Conf mode : ON
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1809
OS Build : 17763.1339
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll PythonScript.dll

**Comment by vinsworldcom** [comment]
> window should be made "modal" as it has no value in being modeless

That was my first thought to "fix" the issue!

**Comment by sasumner** [comment]
@vinsworldcom Yes, but a fix that would result in the _Expected Behavior_ has a lot of value if PR 8780 is not adopted.  We'll "pend" this issue on that PR.

**Comment by vinsworldcom** [comment]
I like the idea of having it in the status bar.  In fact, I see Python script can "override" the status bar.  It'd be a really nice "enhancement" to see a "user-defined space" in the status bar that plugin developers - or scripters - can use to display status or what not without overridding the left most language status bar section as I see it done most often.

But that' taking this a bit off topic - maybe better discussion on the forums.

Cheers.

**Comment by sasumner** [comment]
>...display status or what not without overridding the left most language status bar section as I see it done most often.

Actually, that technique is VERY viable.
I rarely have a need to know that N++ thinks my file type is.
And that is a really LONG field, just ripe for turning it into a custom message. :-)

>It'd be a really nice "enhancement" to see a "user-defined space" in the status bar that plugin developers - or scripters - can use 

I don't want to be pessimistic, but I don't think you'll see this in N++.
